gallery/src/jsUtilPages/WithClass/WithClassExample.scss

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@font-face {
+  font-family: precious;
+  src: url(../../assets/Precious.ttf) format("truetype");
```

Comment:
Is this font OSS?

Reply:
> Is this font OSS?

Here's the license file.  When I read it the first time it seemed like we were fine, but reading it again (particularly the Destribution section), I think including it in a public git repo may be a problem. I'll look for a different one.

> 1001Fonts Free For Commercial Use License (FFC)
>
> Preamble
> In this license, 'Precious' refers to the given .zip file, which may contain one or numerous fonts. These fonts can be of any type (.ttf, .otf, ...) and together they form a 'font family' or in short a 'typeface'.
>
> 1. Copyright
> Precious is the intellectual property of its respective author, provided it is original, and is protected by copyright laws in many parts of the world.
>
> 2. Usage
> Precious may be downloaded and used free of charge for both personal and commercial use. Personal use refers to all usage that does not generate financial income in a business manner, for instance:
>
>  - personal scrapbooking for yourself
>  - recreational websites and blogs for friends and family
>  - prints such as flyers, posters, t-shirts for churches, charities, and non-profit organizations
>
> Commercial use refers to usage in a business environment, including:
>
>  - business cards, logos, advertising, websites for companies
>  - t-shirts, books, apparel that will be sold for money
>  - flyers, posters for events that charge admission
>  - freelance graphic design work
>  - anything that will generate direct or indirect income
>
> 3. Modification
> Precious may not be modified, altered, adapted or built upon without written permission by its respective author. This pertains all files within the downloadable font zip-file.
>
> 4. Distribution
> While Precious may freely be copied and passed along to other individuals for private use as its original downloadable zip-file, it may not be sold or published without written permission by its respective author.
>
> 5. Disclaimer
> Precious is offered 'as is' without any warranty. 1001fonts.com and the respective author of Precious shall not be liable for any damage derived from using this typeface. By using Precious you agree to the terms of this license.

Reply:
> I think including it in a public git repo may be a problem. I'll look for a different one.

451f408a.  This one is released under the Open Font License.

Reply:
> > I think including it in a public git repo may be a problem. I'll look for a different one.
> 
>  This one is released under the Open Font License.

I was thinking that you could have just changed the background colour or something, but this is classier.